YAGSL-Example-main/src/main/java/frc/robot/subsystems/Elevator/ElevatorSubsystem.java

```diff
@@ -89,43 +105,42 @@ private double calculatePid(double position) {
 
     public Command goDown() { // for manual control, sick
         return runOnce(() -> {
-            setpoint = lowestPoint;
+            goToSetpoint(lowestPoint);
```

Comment:
None of these commands calling `goToSetpoint` will do anything. The just call the command but don't return it so they won't actually be scheduled.